public/keymaps/c/coseyfannitutti_romeo_default.json

```
@@ -0,0 +1,20 @@
+{
+  "keyboard": "coseyfannitutti/romeo",
+  "keymap": "default_2a31fbf",
+  "commit": "2a31fbf9a6970ed425c2331f25fb7f92648ffcf1",
+  "layout": "LAYOUT_all",
+  "layers": [
+    [
+      "KC_TAB",  "KC_Q",    "KC_W",    "KC_E",    "KC_R",    "KC_T",    "KC_Y",    "KC_U",    "KC_I",    "KC_O",    "KC_P",    "KC_BSPC",
+      "KC_CAPS", "KC_A",    "KC_S",    "KC_D",    "KC_F",    "KC_G",    "KC_H",    "KC_J",    "KC_K",    "KC_L",    "KC_QUOT",
+      "KC_LSFT", "KC_SLSH", "KC_Z",    "KC_X",    "KC_C",    "KC_V",    "KC_B",    "KC_N",    "KC_M",    "KC_COMM", "KC_DOT",  "KC_RSFT",
+      "KC_LCTL", "KC_LGUI", "KC_LALT",                       "KC_SPC",  "KC_SPC",  "KC_SPC",             "KC_RALT", "MO(1)",   "KC_RCTL",
```

You have an extra comma here.

```suggestion
      "KC_LCTL", "KC_LGUI", "KC_LALT",                       "KC_SPC",  "KC_SPC",  "KC_SPC",             "KC_RALT", "MO(1)",   "KC_RCTL"
```